playbooks/hadoop-build-arm64/run.yaml

```diff
@@ -0,0 +1,32 @@
+- hosts: all
+  become: yes
+  roles:
+    - install-openjdk
+    - install-maven
+  tasks:
+    - name: install required packages
+      shell: |
+        set -exo pipefail
+        apt-get update
+        apt-get install -y build-essential autoconf automake libtool cmake zlib1g-dev pkg-config libssl-dev libsasl2-dev
+        wget https://github.com/protocolbuffers/protobuf/releases/download/v2.5.0/protobuf-2.5.0.tar.bz2
+        wget -O protobuf-2.5.0-arm64.patch https://www.dropbox.com/s/713wql5cw9dfxhx/protobuf-2.5.0-arm64.patch?dl=0
+        tar xvfj protobuf-2.5.0.tar.bz2
+        cd protobuf-2.5.0
+        patch -p1 < ../protobuf-2.5.0-arm64.patch
+        ./autogen.sh
+        ./configure --prefix=/usr
+        make
+        make check
+        make install
+        protoc --version
+        cd ..
+        mkdir -p ~/.m2/repository/io/grpc/protoc-gen-grpc-java/1.15.1/
+        cd ~/.m2/repository/io/grpc/protoc-gen-grpc-java/1.15.1/
+        wget -r -nd -np -k -A 'protoc-*' http://home.apache.org/~aajisaka/repository/io/grpc/protoc-gen-grpc-java/1.15.1/
+        cd '{{ ansible_user_dir }}'/'{{ zuul.project.src_dir }}'
+        mvn package -Pdist -Pdoc -Psrc -Dtar -DskipTests 2>&1 | tee "{{ ansible_user_dir }}/workspace/logs/hadoop_build.log"
```

Comment:
This's a demo, so I think we should show Hadoop trunk compile error in log, we can add a task into the playbook for without aarch64 enabled patch scenairo.
Task order:
1. compile Hadoop trunk, show error in log
2. apply patch
3. compile again, successfully

Reply:
Thanks for the review, as for protobuf, I've already found related issue and asked, they have replied pretty clear their altitude and provided a solution, I'm tring the proposed solution now in all in one env, I will update once I finish succesfully.

Reply:
Thank you.